README.md

```diff
@@ -54,6 +55,11 @@ Check out this [deployment tutorial](https://code.visualstudio.com/tutorials/app
 
     ![Web App Log Stream](resources/Scm_Do_Build_During_Deployment.png)
 
+## Advanced Creation Configuration Settings
+
+* `appService.advancedCreation`
+  * Enables the full control `New Web App Create...` process.  If you don't want Resource Groups, App Service Plans, and WebsiteOS to be smart defaulted, then set this setting to `true`.
```

Comment:
"If you don't want to be smart defaulted" seems a little snarky. It feels like we're saying "If you want to make dumb choices, turn this setting on."

How about something like this:
> Set this to 'true' to explicitly control more settings (i.e. App Service plan size) when creating a web app, rather than using the defaults.

Reply:
🤓 Maybe I was trying to be snarky and dissuade people from using `advancedCreation`.